pkg/asset/installconfig/azure/validation.go

```diff
@@ -526,7 +526,19 @@ func validateNetworks(client API, p *aztypes.Platform, machineNetworks []types.M
 func validateSubnet(client API, fieldPath *field.Path, subnet *aznetwork.Subnet, subnetName string, networks []types.MachineNetworkEntry) field.ErrorList {
 	allErrs := field.ErrorList{}
 
-	subnetIP, _, err := net.ParseCIDR(*subnet.AddressPrefix)
+	var addressPrefix string
+	switch {
+	case subnet.AddressPrefix != nil:
```

Comment:
Is there ever a case where both fields will be filled out?

Reply:
I don't think so, but who knows. The CU has the feature enabled, so they get `addressPrefixes` only. Our dev account doesn't have the feature, so we get `addressPrefix` only. I could not find any azure docs about this.